docs/edit-profile.md

```diff
@@ -1,37 +1,48 @@
-# Edit profile
+# Edit your TechPass profile
 
-TechPass users can edit their TechPass profile anytime by logging into the TechPass portal using non-SE GSIB device.
+TechPass users can edit their TechPass profile anytime by logging into the TechPass portal.
 
-> **Note**
-> Create a [service request](https://go.gov.sg/techpass-sr) with us to update your profile details:
->- If you are a public officer with a SE GSIB device.
->- If you are a vendor or contractor whose TechPass ID's domain is *techpass.gov.sg*.
+?> Users who don't have access to TechPass portal, create a [service request](https://go.gov.sg/techpass-sr) with us to update your profile details.
```

Comment:
Reminder: there's a new service request link that will go live (prod) on May 17th. The new link is https://go.gov.sg/seed-techpass-support

Reply:
yes, Shilpa already shared this. We will update it closer to the date . Thanks!